src/org/centos/pipeline/PackagePipelineUtils.groovy

```diff
@@ -306,21 +321,37 @@ def checkBranch() {
     return result
 }
 
-def packageMetrics() {
-    def tags = [:]
-    def fields = [:]
-    def measurement = env.fed_repo
-    tags['build_result'] = currentBuild.result
-    fields['build_time'] = currentBuild.getDuration().toString()
-
-    return [measurement, tags, fields]
+/**
+ * Parse the repo name from the request list.
+ * Sets env.fed_repo
+ * @param request
+ * @return
+ */
+def repoFromRequest(def request) {
+    if (!env.fed_repo) {
+        try {
+            def pkgUrlTok = request[0].tokenize('/')
+            env.fed_repo = pkgUrlTok.last().tokenize('.')[0]
+        } catch(e) {
+            env.fed_repo = "pkg name unavailable"
+        }
+    }
 }
 
-def pipelineMetrics() {
-    def tags = [:]
-    def fields = [:]
-    def measurement = 'pipeline'
-    tags['build_result'] = currentBuild.result
+/**
+ * Check the fedora version number. Must be fc[2-9][0-9]
+ * @param msgRelease
+ * @return null or fedora release
+ */
+def checkRelease(String msgRelease) {
+    def targetRelease = null
 
-    return [measurement, tags, fields]
-}
+    def release = msgRelease.tokenize('.').last()
+    if (release ==~ /fc[2-9][0-9]/) {
+        targetRelease = release[2, 3]
+    } else {
+        println "Invalid release"
```

Comment:
I think something more along the lines of we don't care about this release would be a little better. Would say it doesn't matter, but if we are storing 500 builds for debugging purposes, might as well be a little more explicit